package.json

```diff
@@ -105,8 +105,8 @@
         "atob": "^2.1.2",
         "formidable": "^1.1.1",
         "getmac": "^1.2.1",
-        "kite-api": "^3.3.0",
-        "kite-connector": "^3.4.0",
+        "kite-api": "=3.3.0",
```

Comment:
This is pinned to 3.5.0 in Atom

Reply:
I pinned it to what was specified here, since that's a known working state.

I can also pin it to the latest released version (3.8/3.9), since that is what was used for today's release. However using the latest released version is actually slightly broken: autostarting Kite doesn't work properly as far as I can tell.

I think I'll just also update this PR to make the fixes so that it works properly with 3.8/3.9.

Reply:
ok, I've pinned it now to 3.9/3.10, and also fixed the call to `runKiteAndWait`